### PR TITLE
Allow admins to retrieve apps via API

### PR DESCRIPTION
**Pull request by amdomanska**
https://github.com/DOAJ/doajPM/issues/3864

### CODE REVIEW

File: portality/api/current/crud/applications.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def retrieve(cls, id, account):
 
         # is the current account the owner of the application
         # if not we raise a 404 because that id does not exist for that user account.
-        if ap.owner != account.id:
+        if "admin" not in account.role and ap.owner != account.id:
```
Comment (richard-jones) [inline portality/api/current/crud/applications.py:200]: Use `account.is_super`

**Review by richard-jones — CHANGES_REQUESTED** [review]
Need to do this with the account API rather than a magic string

File: portality/api/current/crud/applications.py (hunk 1 above)
Comment (amdomanska) [reply portality/api/current/crud/applications.py]: 👍🏼

File: portality/api/current/crud/applications.py [hunk 2]
```diff
@@ -197,7 +197,7 @@ def retrieve(cls, id, account):
 
         # is the current account the owner of the application
         # if not we raise a 404 because that id does not exist for that user account.
-        if ap.owner != account.id:
+        if account.is_super not in account.role and ap.owner != account.id:
```
Comment (richard-jones) [inline portality/api/current/crud/applications.py:200]: I don't think you tested this before committing it.

is_super is a function which determines if the account is the super user.

```
if not account.is_super and ap.owner != account.id
```

**Review by richard-jones — CHANGES_REQUESTED** [review]
Test before you commit code changes :)

File: portality/api/current/crud/applications.py (hunk 2 above)
Comment (amdomanska) [reply portality/api/current/crud/applications.py:200]: > I don't think you tested this before committing it.

I'm afraid you're right. I wanted to push it quickly but got distracted. I'll fix it asap. I'm sorry for that!

**Review by Steven-Eardley — APPROVED** [review]

**Pull request closed without merging**